CMakeLists.txt

```
@@ -59,7 +59,24 @@ if(BUILD_MCNP5 OR BUILD_MCNP6)
   enable_language(Fortran)
 endif()
 
-find_package(MOAB REQUIRED)
+# find_package(MOAB REQUIRED)
+
+if (NOT MOAB_FOUND)
+  include(FetchContent)
+  message(STATUS "MOAB not found. Fetching MOAB...")
+  FetchContent_Declare(
+    moab
+    GIT_REPOSITORY https://bitbucket.org/fathomteam/moab.git
+    GIT_BRANCH master
+    # GIT_TAG        5.5.1
+  )
```

How about we add HDF5 and other cmake args like this and then we can still use fetch
```suggestion
    CMAKE_ARGS
      -DENABLE_HDF5=ON
      -DENABLE_NETCDF=ON
      -DBUILD_SHARED_LIBS=OFF
      -DENABLE_FORTRAN=OFF
      -DENABLE_BLASLAPACK=OFF
  )
```

---

I am not sure if it's a vaild command for FetchContent_Declare, but it is available in ExternalProject_Add. For more: https://discourse.cmake.org/t/fetchcontent-declare-with-cmake-args-does-not-work/11227/2